tree/tree/Controller/Live/LiveViewController.swift

```diff
@@ -0,0 +1,264 @@
+//
+//  LiveViewController.swift
+//  tree
+//
+//  Created by ParkSungJoon on 29/01/2019.
+//  Copyright © 2019 gardener. All rights reserved.
+//
+
+import UIKit
+import Intents
+import NetworkFetcher
+
+class LiveViewController: UIViewController {
+    
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var pageControl: UIPageControl!
```

Comment:
[접근제어]
스토리보드나 Interface Builder 에서 연결한 @IBOutlet, @IBAction 변수 / 메소드에도 알맞은 접근제어자를 설정해주는 것이 좋습니다.
```suggestion
    @IBOutlet private weak var pageControl: UIPageControl!
```